### ISSUE TITLE
Missing Sigmoid when calculating BCE loss / Activation function during inference 

### ISSUE THREAD

**Issue by guozixunnicolas** [jongwook/onsets-and-frames#35]
Hi JongWook,

Thank you for your implementation. 

I have 2 questions regarding your implementation. 

1. Based on the [documentation](https://pytorch.org/docs/stable/generated/torch.nn.functional.binary_cross_entropy.html), a sigmoid activation function needs to be applied to the logits before calculating the BCE loss. However, in transcriber.py, a sigmoid activation is not applied before activation. May I know whether this is an error? 

2. During inference in evaluate.py, I noticed that a relu activation is applied in the function evaluate(). Instead of relu, shouldn't sigmoid be used here? 

Best,

Nicolas

**Comment by guozixunnicolas** [comment]
Finished my investigation: 1. The Sigmoid activation is embedded within the conv stacks. It's correct to use BCE loss. 2. The Relu activation in evaluate() would not make a difference since value.squeeze_(0).relu_() would not change the pred[key] in the dictionary anyway. Although it's a bit confusing..
